### PR TITLE
#570: invite inside of start() and remove one invitation in test

### PR DESCRIPTION
**Pull request by lautarobock**
#570: invite inside of start() and remove one invitation in test

### CODE REVIEW

**Comment by karato** [comment]
@lautarobock Thanks for your pull request, let me find someone who can review it

**Comment by karato** [comment]
@carlosmiranda review it

**Comment by carlosmiranda** [comment]
@rultor seems good to merge already

**Comment by dmzaytsev** [comment]
@lautarobock what about a test first?
I'm talking about a new test for `MkInbox` which should to check for the invited friend.
Please implement it.

**Comment by lautarobock** [comment]
@dmzaytsev ready

**Comment by dmzaytsev** [comment]
@lautarobock thanks!

@carlosmiranda please review the new code

File: netbout-web/src/test/java/com/netbout/email/EmAliasTest.java [hunk 1]
```diff
@@ -58,7 +58,6 @@ public void sendsEmail() throws Exception {
         final MkBase base = new MkBase();
         final Alias alias = new EmAlias(base.randomAlias(), postman);
         final Bout bout = alias.inbox().bout(alias.inbox().start());
-        bout.friends().invite(base.randomAlias().name());
```
Comment (carlosmiranda) [inline netbout-web/src/test/java/com/netbout/email/EmAliasTest.java]: @lautarobock This test is failing on Travis, please check.

**Comment by carlosmiranda** [comment]
@lautarobock Build is failing, please fix.

**Comment by dmzaytsev** [comment]
@lautarobock please merge with master

**Comment by lautarobock** [comment]
@dmzaytsev done

**Comment by carlosmiranda** [comment]
@rultor we can merge now.

**Comment by dmzaytsev** [comment]
@rultor merge please

**Comment by karato** [comment]
@ypshenychka please, review this ticket for compliance with our [QA rules](http://www.teamed.io/qa.html)

**Comment by karato** [comment]
@rultor deploy pls

**Comment by ypshenychka** [comment]
@carlosmiranda 
According to our [QA Rules](http://www.teamed.io/qa.html):
>The code reviewer found at least three problems in the code.

Try to find at least three problems while future reviews. If the PR is too small it should be combined with other related issue and fixed. It would be a high quality ticket.

Please confirm that everything is clear and you have no questions regarding this approach.

**Comment by carlosmiranda** [comment]
@ypshenychka all is clear, I have no questions.

**Comment by ypshenychka** [comment]
@carlosmiranda thanks

**Comment by ypshenychka** [comment]
@karato Quality is acceptable.

**Comment by karato** [comment]
> @karato Quality is acceptable.

@ypshenychka thanks for the QA review, we'll work better next time

**Comment by lautarobock** [comment]
@ypshenychka thanks for the review. Would you tell me why is `acceptable`? just for further improvements

**Comment by ypshenychka** [comment]
@lautarobock The reason is that code reviewer have not revealed at least three problems in the code (according to [QA Rules](http://www.teamed.io/qa.html)).

**Comment by lautarobock** [comment]
@ypshenychka Thanks!

**Comment by karato** [comment]
@carlosmiranda I added 10 mins to @ypshenychka (for QA review) in transaction 58185589. Much appreciated! **19 mins** was added to your account, payment ID `58185596`, time spent is 87 hours and 44 mins. you're getting extra minutes here (c=4). +19 added to your rating, current score is: [+4250](http://www.netbout.com/b/35023?open=rating)

**Pull request merged**